modules/presentation.py

```diff
@@ -164,7 +164,7 @@ def pie_chart(df, title, save=None):
     plt.ylabel("")
 
     if save:
-        save_plot(plt, save)
+        save_plot(plt, title)
```

Comment:
I misunderstood the use of the save parameter. Thought it was meant to be used as a boolean. Should be changed back